src/net.cpp

```diff
@@ -1648,6 +1648,14 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
 
             if (add_fixed_seeds_now) {
                 std::vector<CAddress> seed_addrs{ConvertSeeds(Params().FixedSeeds())};
+                // We will not make outgoing connections to peers that are unreachable
```

Comment:
Maybe it wants:
```cpp
/* Blah
 * blah
 * blah */
```

Reply:
No, there are two random funcional tests failing in each failed CI run for some reason (different one in each case), should not be related with this change.

Reply:
The second one is new to me, I opened #26051 - but yes, obviously unrelated.